src/iotjs_module_buffer.cpp

```diff
@@ -82,6 +82,17 @@ size_t BufferWrap::length() {
 }
 
 
+static size_t BoundRange(int index, size_t low, size_t upper) {
```

Comment:
No. Grep for 'static void'  and you can see this is the correct style :) Did you find any example of the lowercase style? Is it mixed in the actual code base?